src/components/AppSpinner.tsx

```diff
@@ -1,17 +1,31 @@
 import React from "react";
-import { Row } from "nav-frontend-grid";
-import { Loader } from "@navikt/ds-react";
+import { Loader, LoaderProps } from "@navikt/ds-react";
+import styled from "styled-components";
 
-const AppSpinner = () => {
+const SpinnerRow = styled.div<{ isSmall?: boolean }>`
+  display: flex;
+  justify-content: center;
+  margin-bottom: ${(props) => (props.isSmall ? "0.5em" : "4em")};
+  > * {
+    &:not(:last-child) {
+      margin-right: 0.25em;
+    }
+  }
+`;
+
+interface AppSpinnerProps {
+  size?: LoaderProps["size"];
+}
+const AppSpinner = ({ size = "2xlarge" }: AppSpinnerProps) => {
```

Comment:
Her tror jeg man kan bruke `ComponentProps<typeof Loader>` i stedet for å definere våre egne `AppSpinnerProps`. Og samtidig bruke `{ size = "2xlarge", ...rest }` og sende `...rest` videre til `Loader`.

Reply:
Eller så kunne man bare brukt Loader direkte i vedlegg-link-komponenten i stedet for å tilpasse denne. AppSpinner er kanskje tenkt brukt ved lasting av en hel side (ref aria-label)?

Reply:
Prøvde meg på en extend av Loader først, men det funket ikke helt 🤔 
Men ComponentProps hadde jeg ikke hørt om, det funket veldig bra! Kult 🚀 
https://www.totaltypescript.com/react-component-props-type-helper

Reply:
Så ikke andre kommentaren før nå 🤔 Hmm, ja den er kanskje brukt for å laste hele sider, godt poeng egentlig...

Reply:
Fikset slik at man bruker `Loader` direkte i stedet nå.